### DISCUSSION TITLE
Logic can't see AG plugin

### DISCUSSION THREAD

**Discussion by kaczysko** [apohl79/audiogridder#1070] [Help]
Hi, im on Mac M1 pro Monterey 12.6 with logic pro 10.7.4, and logic doesn't see audiogridder plugin. works on reaper. any thougts?

**Comment by CaBuffalo** [comment]
It works on my Intel Mac.

Did you see it in the plugin manager?

On Fri, Oct 21, 2022 at 2:13 AM kaczysko ***@***.***> wrote:

> Hi, im on Mac M1 pro Monterey 12.6 with logic pro 10.7.4, and logic
> doesn't see audiogridder plugin. works on reaper. any thougts?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/1070>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6C5HEGIMYPX7PQDLG3WEJNCNANCNFSM6AAAAAARK67OU4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by kaczysko** [reply]
no it don't shows there. but to be 100% sure, what name it's got there and in what manufacturer?

**Reply by kaczysko** [reply]
also i did restart my mac and clear au cache to force logic to re-search for au plugins. still nothing

**Comment by CaBuffalo** [comment]
It might be caused by your M1 processor.

On Fri, Oct 21, 2022 at 12:06 PM kaczysko ***@***.***> wrote:

> also i did restart my mac and clear au cache to force logic to re-search
> for au plugins. still nothing
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/1070#discussioncomment-3936547>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AWZ7OHHHUTD6ZEHR3WELSRRANCNFSM6AAAAAARK67OU4>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by kaczysko** [reply]
i also think it can be that matter but i see at some other forums that people says they got it running on logic on m1.

**Reply by apohl79** [reply]
have you installed the arm or universal build?

**Reply by kaczysko** [reply]
arm.

**Reply by apohl79** [reply]
You can try the universal build. Are you running logic under rosetta?

**Reply by kaczysko** [reply]
yes under rosetta. i got halfly succeed, logic see the universal build pluigin. still, i can't connect with the server. its on parallel windows on my mac, i tried put several IP adresses (127. etc and those in windows network preferences) but it sees that the plugin sees the server (i change its name and ID in AGserver app in windows) but still it says not connected. firewalls inactive.

**Reply by apohl79** [reply]
Well, under rosetta you are in x86 land. But the universal build contains binaries for both architectures.

Regarding your networking issue, try to create a host only interface in parallels and make it a private network in windows.

**Reply by kaczysko** [reply]
all right. i tried several installers of win 10 instead of 11 but none of them works, so i installed win 11 again (pararell desktop default) installed only server on win and only plugin on osx and it WORKS!!!! thanks for your help!!!
